helm/policy-meta-operator/templates/deployment.yaml

```diff
@@ -50,9 +50,27 @@ spec:
         imagePullSecrets:
           {{- toYaml . | nindent 8 }}
       {{- end }}
+      volumes:
+        - name: edgedb-tls
+          secret:
+            secretName: {{ .Values.edgedb.tlsSecret.name }}
       containers:
       - name: {{ include "resource.default.name" . }}
         image: "{{ default .Values.image.registry (include "global.imageRegistry" . ) }}/{{ .Values.image.name }}:{{ .Values.image.tag | default .Chart.Version }}"
+        env:
+        - name: EDGEDB_HOST
+          value: {{ .Values.edgedb.host }}
+        - name: EDGEDB_PORT
+          value: {{ .Values.edgedb.port }}
+        - name: EDGEDB_USER
+          value: {{ .Values.edgedb.user }}
+        - name: EDGEDB_PASSWORD
+          valueFrom:
+           secretKeyRef:
+              name: {{ .Values.edgedb.passwordSecret.name }}
+              key: password
```

Comment:
Even better would be to read the file directly so that the password is never in the environment and the client can be reloaded if the secret changes. This is fine to get started though

Reply:
I will just make sure this works and then move to file, is an easy change

Reply:
Done, bam. Had to do a little trick because there is no env for password file.